modules/api/main.tf

```diff
@@ -418,6 +418,56 @@ DEFINITION
 
 }
 
+resource "aws_ecs_task_definition" "api_exec_task_definition" {
+  family             = "nerves-hub-${terraform.workspace}-api-exec"
+  task_role_arn      = aws_iam_role.api_task_role.arn
+  execution_role_arn = var.task_execution_role.arn
+
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = var.cpu
+  memory                   = var.memory
+
+  tags = var.tags
+
+  container_definitions = <<DEFINITION
+   [
+     ${module.logging_configs.fire_lens_container},
+     ${module.logging_configs.datadog_container},
+     {
+       "portMappings": [
```

Comment:
Do we need ports open for a remote console? I don't think so... but happy to be overruled.

Reply:
> I think theres a few more changes we'll need to make here, off the top of my head permissions to enable this on the cluster or service need to be enabled https://aws.amazon.com/blogs/containers/new-using-amazon-ecs-exec-access-your-containers-fargate-ec2/ Also we should make a `remote-console.sh` script https://github.com/smartrent/control.smartrent.com/blob/main/infrastructure/scripts/remote-console.sh or do you think that will go in another repo?
> 
> Also could you add the ticket number on the front of this PR so I can find it in Jira 🙇

updated the name didnt see that thank you, and Ill take a look at the permissions and yeah as for the remote console script its going in this repo here: https://github.com/smartrent/nerves_hub_infrastructure/pull/131

Reply:
removed the ports and resolved all the issues with this. Permissions are updated as well and tested for all 3 services